main.py

```diff
@@ -124,45 +138,129 @@ def check(m):
             users = await reaction.users().flatten()
             if len(users) == 1:
                 return await msg.edit(embed=discord.Embed(title="Nobody has won the giveaway."))
+                
     try:
         winners = random.sample([user for user in users if not user.bot], k=winerscount)
     except ValueError:
         return await giveawaychannel.send("not enough participants")
+    
     winnerstosend = "\n".join([winner.mention for winner in winners])
+    
+    
 
-    win = await msg.edit(embed=discord.Embed(title="WINNER",
-                                             description=f"Congratulations {winnerstosend}, you have won **{msg4.content}**!",
-                                             color=discord.Color.blue()))
 
+    win = await msg.edit(embed = discord.Embed(title = "WINNER" , description = f"Congratulations {winnerstosend}, you have won **{msg4.content}**!" , color = discord.Color.blue()))
+    
+    print(Fore.GREEN + winnerstosend , Fore.WHITE + "has won: " , Fore.GREEN + msg4.content)

```

Comment:
I don't think we need a prints here

Reply:
It's just to make the terminal look better, all github open source over does it.

Reply:
If you host bot on VPS you will not see prints

Reply:
it's alright then vps or not, if they have it it will show I guess.